### PR TITLE
Add v0.1.0 release notes

### LINKED ISSUE #49: Release v0.1.0

### PR DESCRIPTION
**Pull request by denkensk**
Fix #49

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Kube-queue Release Notes
+
+## v0.1.0
+### Features
+- Build a framework to support different
+  queueing policies and quota systems
+- Support two basic queueing policies for the elements in single queue: FIFO、Priority
+- Support dynamic adjustment of job priority in queue
+- Integrate with the ResourceQuota in Kubernetes
+- Integrate with TF-operator、Pytorch-operator for distribute deeplearning training
```
Comment (zw0610) [inline CHANGELOG.md:10]: Could we change `TF-operator` and `Pytorch-operator` to `TFJob` and `PyTorchJob`? Also, change `、` to `, `

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Kube-queue Release Notes
+
+## v0.1.0
+### Features
+- Build a framework to support different
+  queueing policies and quota systems
```
Comment (zw0610) [inline CHANGELOG.md:6]: why new line?

Comment (denkensk) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (denkensk) [reply CHANGELOG.md]: Done

**Pull request merged**